forward/testing/spike_test.lua

```diff
@@ -0,0 +1,67 @@
+local ffi = require("ffi")
+
+local B = require("apps.basic.basic_apps")
+local P = require("apps.pcap.pcap")
+local C = require("ffi").C
+local Ethernet = require("lib.protocol.ethernet")
+local IPV4 = require("lib.protocol.ipv4")
+local link = require("core.link")
+local packet = require("core.packet")
+
+local Rewriting = require("rewriting")
+local godefs = require("godefs")
+
+local packet_synthesis = require("testing/packet_synthesis")
+local TestStreamApp = require("testing/test_stream_app")
+
+local function runmain()
+   godefs.Init()
+   godefs.AddBackend("http://cheesy-fries.mit.edu/health",
+                     IPV4:pton("1.3.5.7"), 4)
+   godefs.AddBackend("http://strawberry-habanero.mit.edu/health",
+                     IPV4:pton("2.4.6.8"), 4)
+   C.usleep(3000000) -- wait for backends to come up
+
+   local spike_mac = "00:00:00:00:00:00"
```

Comment:
I think a MAC address of all zeros is special somehow.  I don't think it matters for us, but as a matter of good form we should probably just use some more realistic MACs (you can look up the prefixes for real manufacturers).